Source/WebDriver/SessionHost.h

```diff
@@ -97,7 +97,7 @@ class SessionHost final
     long sendCommandToBackend(const String&, RefPtr<JSON::Object>&& parameters, Function<void (CommandResponse&&)>&&);
 
 #if ENABLE(WEBDRIVER_BIDI)
-    void addEventHandler(WeakPtr<BiDiEventHandler>&& handler) { m_eventHandler = WTFMove(handler); }
+    void addBidiHandler(WeakPtr<BidiMessageHandler>&& handler) { m_bidiHandler = WTFMove(handler); }
```

Comment:
If there is only one, then it should be `setBidiHandler`. I'd prefer `setBidiMessageHandler` to match the type.